charts/datadog/templates/datadog-yaml-configmap.yaml

```diff
@@ -30,7 +30,6 @@ data:
       enabled: true
       apm_non_local_traffic: true
       max_memory: 0
-      max_cpu_percent: 0
```

Comment:
Why remove the default?

Reply:
Ah I see that you explain this a bit in the other PR. Just worth a call out in this PR description too.